tools/checkout-deps.sh

```diff
@@ -34,11 +34,9 @@ elif [ `uname` != "Linux" ] && [ -n "${COMSPEC:+1}" ]; then
   decomp=unzip
 fi
 
-if [ ! -d "sourcemod" ]; then
-  if [ ! -d "sourcemod-1.5" ]; then
```

Comment:
This if statement seemed like an old redundant edge case so I've removed it, can be added back if necessary